utils.py

```diff
@@ -724,6 +724,70 @@ def get_hashable_value(value):
         return value
 
 
+def convert_states(states_dict, conversion_fn):
```

Comment:
This is in the wrong place. Stuff in utils.py should not have anything to do with explorations.

I think perhaps you want something along the lines of State.convert_html_fields(state_dict) as a classmethod/staticmethod (in exp_domain.py).

Reply:
Done!